src/plugins/discover/public/application/main/components/top_nav/discover_topnav.tsx

```diff
@@ -114,6 +196,8 @@ export const DiscoverTopNav = ({
       showSaveQuery={!!services.capabilities.discover.saveQuery}
       showSearchBar={true}
       useDefaultBehaviors={true}
+      dataViewPickerComponentProps={dataViewPickerProps}
+      displayStyle="detached"
```

Comment:
@stratoula I can't figure out why this prop `displayStyle` in Discover's instance is not being passed down to the SearchBar 😕

Reply:
Fixed https://github.com/elastic/kibana/pull/128401/commits/6867bea617e71eb01d4489c6b0146651f60d95cc